packages/cli/src/services/user.service.ts

```diff
@@ -120,6 +120,7 @@ export class UserService {
 
 		let publicUser: PublicUser = {
 			...rest,
+			globalScopes: user.globalScopes,
```

Comment:
Should we add a test to this? To make sure we're always returning this property at least.

Reply:
I've added a quick check to the auth tests. These may need expanded on in the future though.